components/vf-form/vf-form__helper/vf-form__helper.scss

```diff
@@ -13,10 +13,10 @@
   @include set-type(text-body--5, $custom-margin-bottom: 0);
 
   color: set-color(vf-color--grey);
+}
+
 
-  .vf-form__floatlabel & {
-    left: 24px;
-    position: relative;
-    top: 12px;
-  }
+.vf-form__helper--error {
```

Comment:
Should we add any CSS for form element `:invalid` states ... which I guess would be turning the border red or something? (perhaps similar to `vf-form__input:disabled`)

Reply:
good idea - will add